### PR TITLE
pythonPackages.*: Disable various checkPhases with zero tests run, libgda: Fix building with newest mysql, vtk_{7,8}: Fix compiling

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/clifford/default.nix [hunk 1]
```diff
@@ -22,12 +22,15 @@ buildPythonPackage rec {
...
-    url = "https://github.com/pygae/clifford/pull/388/commits/955d141662c68d3d61aa50a162b39e656684c208.patch";
-    sha256 = "00m8ias58xycn5n78sy9wywf4wck1v0gb8gzmg40inzdiha93jyz";
-  }) ];
+
+  patches = [
+    (fetchpatch {
+      # Compatibility with h5py 3.
+      # Will be included in the next releasse after 1.3.1
+      url = "https://github.com/pygae/clifford/pull/388/commits/955d141662c68d3d61aa50a162b39e656684c208.patch";
+      sha256 = "0pkpwnk0kfdxsbzsxqlqh8kgif17l5has0mg31g3kyp8lncj89b1";
```
Comment (cole-h) [inline pkgs/development/python-modules/clifford/default.nix:31]: Pretty sure this hash is wrong.

```
$ nix-prefetch-url 'https://github.com/pygae/clifford/commit/955d141662c68d3d61aa50a162b39e656684c208.patch'
path is '/nix/store/ya0hpzdm4kd6gaxx0abv7hn1xcadinq0-955d141662c68d3d61aa50a162b39e656684c208.patch'
00m8ias58xycn5n78sy9wywf4wck1v0gb8gzmg40inzdiha93jyz
```

The new hash you've pasted is the hash of the `.diff`, not the hash of the `.patch`.

**Review by cole-h — CHANGES_REQUESTED** [review]
Even if there are no tests at this point in time, I don't think the `checkPhase` should be disabled. If upstream ever adds tests, we now have to remove the `doCheck = false;` line, rather than just bumping the version (and maybe disabling certain tests that don't work).

It takes an extra bit of time to figure out it has no tests and just move on to the next phase. Maybe it's just me, but I don't think the disabling of the `checkPhase` is necessary or desired.

File: pkgs/development/python-modules/clifford/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/clifford/default.nix:31]: Yeah, because the patch is fetched not the commit...

**Comment by SuperSandro2000** [comment]
> Even if there are no tests at this point in time, I don't think the `checkPhase` should be disabled. If upstream ever adds tests, we now have to remove the `doCheck = false;` line, rather than just bumping the version (and maybe disabling certain tests that don't work).

It is a maintainers duty to keep up with this. Also most of the time we would need to add ``pytestCheckHook`` anyway because ``python setup.py test`` is going away and planned to be disabled by default in the future because many pythonApplications who do not have any tests also execute it.

Also a maintainers duty is it to check if all dependencies are still required and the package works correctly. Also python is the only place where we execute those tests by default which most of the time do not reveal bugs in our packaging but code bugs. If we are doing pythomImportsCheck everywhere it is enough for the simpler packages. Complicated things like tensorflow or pandas require tests.

> It takes an extra bit of time to figure out it has no tests and just move on to the next phase. Maybe it's just me, but I don't think the disabling of the checkPhase is necessary or desired.

I didn't come up with it but jonringer and I think he knows what he is doing. If we update something I don't want to deal with failing tests which where not there before.

File: pkgs/development/python-modules/clifford/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/clifford/default.nix:31]: For me this is the new hash because something probably changed or the patch got rebased.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108799` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>29 packages failed to build and are new build failures:</summary>
  <ul>
<li>appcsxcad: log was empty</li>
<li>cq-editor: log was empty</li>
<li>gtkd: log was empty</li>
<li>horizon-eda: log was empty</li>
<li>inkcut: log was empty</li>
<li>inkscape-extensions.inkcut: log was empty</li>
<li>libsForQt5.qcsxcad: log was empty</li>
<li>(libsForQt515.qcsxcad): log was empty</li>
<li>libsForQt512.qcsxcad: log was empty</li>
<li>libsForQt514.qcsxcad: log was empty</li>
<li>openems: log was empty</li>
<li>python37Packages.cadquery: log was empty</li>
<li>python37Packages.enamlx: log was empty</li>
<li>python37Packages.python-csxcad: log was empty</li>
<li>python37Packages.python-openems: log was empty</li>
<li>python37Packages.pythonocc-core: log was empty</li>
<li>python37Packages.vtk_8: log <a href="https://termbin.com/gzzq">https://termbin.com/gzzq</a></li>
<li>python38Packages.enamlx: log was empty</li>
<li>python38Packages.python-csxcad: log was empty</li>
<li>python38Packages.python-openems: log was empty</li>
<li>python38Packages.pythonocc-core: log was empty</li>
<li>python38Packages.vtk_8: log <a href="https://termbin.com/yqlb">https://termbin.com/yqlb</a></li>
<li>python39Packages.enamlx: log was empty</li>
<li>python39Packages.python-csxcad: log was empty</li>
<li>python39Packages.python-openems: log was empty</li>
<li>python39Packages.pythonocc-core: log was empty</li>
<li>python39Packages.vtk_8: log <a href="https://termbin.com/4jpt">https://termbin.com/4jpt</a></li>
<li>simpleitk: log <a href="https://termbin.com/9dvz">https://termbin.com/9dvz</a></li>
<li>tilix: log was empty</li>
  </ul>
</details>
<details>
<summary>
60 packages built:
</summary>
<ul>
<li>ants</li>
<li>c3d</li>
<li>csxcad</li>
<li>datasette (python38Packages.datasette)</li>
<li>dockbarx</li>
<li>elastix</li>
<li>ezminc</li>
<li>gdcm</li>
<li>glom</li>
<li>gnome3.anjuta</li>
<li>libgda (gnome3.libgda)</li>
<li>gtkpod</li>
<li>gtranslator</li>
<li>i3altlayout</li>
<li>itk</li>
<li>itk4</li>
<li>libgdamm</li>
<li>mirtk</li>
<li>mitmproxy (python38Packages.mitmproxy)</li>
<li>mrtrix</li>
<li>pantheon.elementary-music</li>
<li>python37Packages.asgiref</li>
<li>python37Packages.channels</li>
<li>python37Packages.channels-redis</li>
<li>python37Packages.clifford</li>
<li>python37Packages.daphne</li>
<li>python37Packages.datasette</li>
<li>python37Packages.dicom2nifti</li>
<li>python37Packages.django_3</li>
<li>python37Packages.gdcm</li>
<li>python37Packages.mayavi</li>
<li>python37Packages.mitmproxy</li>
<li>python37Packages.nixpkgs</li>
<li>python37Packages.vtk (python37Packages.vtk_7)</li>
<li>python38Packages.asgiref</li>
<li>python38Packages.channels</li>
<li>python38Packages.channels-redis</li>
<li>python38Packages.clifford</li>
<li>python38Packages.daphne</li>
<li>python38Packages.dicom2nifti</li>
<li>python38Packages.django_3</li>
<li>python38Packages.gdcm</li>
<li>python38Packages.mayavi</li>
<li>python38Packages.nixpkgs</li>
<li>python38Packages.vtk (python38Packages.vtk_7)</li>
<li>python39Packages.asgiref</li>
<li>python39Packages.channels</li>
<li>python39Packages.channels-redis</li>
<li>python39Packages.daphne</li>
<li>python39Packages.dicom2nifti</li>
<li>python39Packages.django_3</li>
<li>python39Packages.gdcm</li>
<li>python39Packages.mitmproxy</li>
<li>python39Packages.nixpkgs</li>
<li>python39Packages.vtk (python39Packages.vtk_7)</li>
<li>raiseorlaunch</li>
<li>remarkable-mouse</li>
<li>sequeler</li>
<li>urlwatch</li>
<li>xfce.xfce4-dockbarx-plugin (xfce.xfce4_dockbarx_plugin)</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108799` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
6 packages marked as broken and skipped:
</summary>
<ul>
<li>cq-editor</li>
<li>gnome3.libgda</li>
<li>gtkd</li>
<li>libgda</li>
<li>python39Packages.datasette</li>
<li>remarkable-mouse</li>
</ul>
</details>
<details>

  <summary>32 packages failed to build and are new build failures:</summary>
  <ul>
<li>ants: log was empty</li>
<li>c3d: log was empty</li>
<li>elastix: log was empty</li>
<li>ezminc: log was empty</li>
<li>gdcm: log was empty</li>
<li>inkcut: log was empty</li>
<li>itk4: log was empty</li>
<li>mnemosyne: log <a href="https://termbin.com/uxgz">https://termbin.com/uxgz</a></li>
<li>python37Packages.cadquery: log was empty</li>
<li>python37Packages.dicom2nifti: log was empty</li>
<li>python37Packages.enamlx: log was empty</li>
<li>python37Packages.gdcm: log was empty</li>
<li>python37Packages.mayavi: log was empty</li>
<li>python37Packages.pythonocc-core: log was empty</li>
<li>python37Packages.vtk: log <a href="https://termbin.com/xvhq">https://termbin.com/xvhq</a></li>
<li>(python37Packages.vtk_7): log was empty</li>
<li>python37Packages.vtk_8: log <a href="https://termbin.com/khg1">https://termbin.com/khg1</a></li>
<li>python38Packages.dicom2nifti: log was empty</li>
<li>python38Packages.enamlx: log was empty</li>
<li>python38Packages.gdcm: log was empty</li>
<li>python38Packages.mayavi: log was empty</li>
<li>python38Packages.pythonocc-core: log was empty</li>
<li>python38Packages.vtk: log <a href="https://termbin.com/ilno">https://termbin.com/ilno</a></li>
<li>(python38Packages.vtk_7): log was empty</li>
<li>python38Packages.vtk_8: log <a href="https://termbin.com/nmsz">https://termbin.com/nmsz</a></li>
<li>python39Packages.dicom2nifti: log was empty</li>
<li>python39Packages.enamlx: log was empty</li>
<li>python39Packages.gdcm: log was empty</li>
<li>python39Packages.pythonocc-core: log was empty</li>
<li>python39Packages.vtk: log <a href="https://termbin.com/9mur">https://termbin.com/9mur</a></li>
<li>(python39Packages.vtk_7): log was empty</li>
<li>python39Packages.vtk_8: log <a href="https://termbin.com/h0td">https://termbin.com/h0td</a></li>
  </ul>
</details>
<details>
<summary>
44 packages built:
</summary>
<ul>
<li>anki</li>
<li>appdaemon</li>
<li>datasette (python38Packages.datasette)</li>
<li>gdbgui</li>
<li>mitmproxy (python38Packages.mitmproxy)</li>
<li>python37Packages.asgiref</li>
<li>python37Packages.asyncio-dgram</li>
<li>python37Packages.channels</li>
<li>python37Packages.channels-redis</li>
<li>python37Packages.clifford</li>
<li>python37Packages.daphne</li>
<li>python37Packages.datasette</li>
<li>python37Packages.django_3</li>
<li>python37Packages.flask-socketio</li>
<li>python37Packages.mitmproxy</li>
<li>python37Packages.nixpkgs</li>
<li>python37Packages.python-engineio</li>
<li>python37Packages.python-socketio</li>
<li>python37Packages.pywizlight</li>
<li>python38Packages.asgiref</li>
<li>python38Packages.asyncio-dgram</li>
<li>python38Packages.channels</li>
<li>python38Packages.channels-redis</li>
<li>python38Packages.clifford</li>
<li>python38Packages.daphne</li>
<li>python38Packages.django_3</li>
<li>python38Packages.flask-socketio</li>
<li>python38Packages.nixpkgs</li>
<li>python38Packages.python-engineio</li>
<li>python38Packages.python-socketio</li>
<li>python38Packages.pywizlight</li>
<li>python39Packages.asgiref</li>
<li>python39Packages.asyncio-dgram</li>
<li>python39Packages.channels</li>
<li>python39Packages.channels-redis</li>
<li>python39Packages.daphne</li>
<li>python39Packages.django_3</li>
<li>python39Packages.flask-socketio</li>
<li>python39Packages.mitmproxy</li>
<li>python39Packages.nixpkgs</li>
<li>python39Packages.python-engineio</li>
<li>python39Packages.python-socketio</li>
<li>python39Packages.pywizlight</li>
<li>urlwatch</li>
</ul>
</details>

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -16673,13 +16673,15 @@ in
   vte_290 = callPackage ../development/libraries/vte/2.90.nix { };
 
   vtk_7 = libsForQt515.callPackage ../development/libraries/vtk/7.x.nix {
+    stdenv = gcc9Stdenv;
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:16676]: There are no releases for 3 years and there is probably no fix with gcc10.

File: pkgs/top-level/all-packages.nix [hunk 3]
```diff
@@ -16673,13 +16673,15 @@ in
...
     inherit (darwin) libobjc;
     inherit (darwin.apple_sdk.libs) xpc;
     inherit (darwin.apple_sdk.frameworks) Cocoa CoreServices DiskArbitration
                                           IOKit CFNetwork Security ApplicationServices
                                           CoreText IOSurface ImageIO OpenGL GLUT;
   };
   vtk_8 = libsForQt515.callPackage ../development/libraries/vtk/8.x.nix {
+    stdenv = gcc9Stdenv;
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:16684]: There are no releases for a year for 8.X and there is probably no fix with gcc10 and I couldn't find a branch for it.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108799` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>31 packages failed to build and are new build failures:</summary>
  <ul>
<li>ants: log was empty</li>
<li>c3d: log was empty</li>
<li>elastix: log was empty</li>
<li>ezminc: log was empty</li>
<li>gdcm: log was empty</li>
<li>inkcut: log was empty</li>
<li>inkscape-extensions.inkcut: log was empty</li>
<li>itk: log was empty</li>
<li>itk4: log was empty</li>
<li>mnemosyne: log <a href="https://termbin.com/scc9">https://termbin.com/scc9</a></li>
<li>python37Packages.cadquery: log was empty</li>
<li>python37Packages.dicom2nifti: log was empty</li>
<li>python37Packages.enamlx: log was empty</li>
<li>python37Packages.gdcm: log was empty</li>
<li>python37Packages.mayavi: log was empty</li>
<li>python37Packages.pythonocc-core: log was empty</li>
<li>python37Packages.vtk: log <a href="https://termbin.com/3km6">https://termbin.com/3km6</a></li>
<li>python37Packages.vtk_8: log <a href="https://termbin.com/w2jw">https://termbin.com/w2jw</a></li>
<li>python38Packages.dicom2nifti: log was empty</li>
<li>python38Packages.enamlx: log was empty</li>
<li>python38Packages.gdcm: log was empty</li>
<li>python38Packages.mayavi: log was empty</li>
<li>python38Packages.pythonocc-core: log was empty</li>
<li>python38Packages.vtk: log <a href="https://termbin.com/wfmj">https://termbin.com/wfmj</a></li>
<li>python38Packages.vtk_8: log <a href="https://termbin.com/nvmq">https://termbin.com/nvmq</a></li>
<li>python39Packages.dicom2nifti: log was empty</li>
<li>python39Packages.enamlx: log was empty</li>
<li>python39Packages.gdcm: log was empty</li>
<li>python39Packages.pythonocc-core: log was empty</li>
<li>python39Packages.vtk: log <a href="https://termbin.com/eqmk">https://termbin.com/eqmk</a></li>
<li>python39Packages.vtk_8: log <a href="https://termbin.com/1v4k">https://termbin.com/1v4k</a></li>
  </ul>
</details>
<details>
<summary>
24 packages built:
</summary>
<ul>
<li>anki</li>
<li>appdaemon</li>
<li>gdbgui</li>
<li>python37Packages.asyncio-dgram</li>
<li>python37Packages.clifford</li>
<li>python37Packages.flask-socketio</li>
<li>python37Packages.nixpkgs</li>
<li>python37Packages.python-engineio</li>
<li>python37Packages.python-socketio</li>
<li>python37Packages.pywizlight</li>
<li>python38Packages.asyncio-dgram</li>
<li>python38Packages.clifford</li>
<li>python38Packages.flask-socketio</li>
<li>python38Packages.nixpkgs</li>
<li>python38Packages.python-engineio</li>
<li>python38Packages.python-socketio</li>
<li>python38Packages.pywizlight</li>
<li>python39Packages.asyncio-dgram</li>
<li>python39Packages.flask-socketio</li>
<li>python39Packages.nixpkgs</li>
<li>python39Packages.python-engineio</li>
<li>python39Packages.python-socketio</li>
<li>python39Packages.pywizlight</li>
<li>urlwatch</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108799` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>14 packages failed to build and are new build failures:</summary>
  <ul>
<li>appcsxcad: log was empty</li>
<li>gtkd: log was empty</li>
<li>qcsxcad: log was empty</li>
<li>(libsForQt5.qcsxcad): log was empty</li>
<li>libsForQt512.qcsxcad: log was empty</li>
<li>libsForQt514.qcsxcad: log was empty</li>
<li>openems: log was empty</li>
<li>python37Packages.python-csxcad: log was empty</li>
<li>python37Packages.python-openems: log was empty</li>
<li>python38Packages.python-csxcad: log was empty</li>
<li>python38Packages.python-openems: log was empty</li>
<li>python39Packages.python-csxcad: log was empty</li>
<li>python39Packages.python-openems: log was empty</li>
<li>tilix: log was empty</li>
  </ul>
</details>
<details>
<summary>
53 packages built:
</summary>
<ul>
<li>ants</li>
<li>c3d</li>
<li>cq-editor</li>
<li>csxcad</li>
<li>dockbarx</li>
<li>elastix</li>
<li>ezminc</li>
<li>gdcm</li>
<li>glom</li>
<li>gnome3.anjuta</li>
<li>gnome3.libgda</li>
<li>gtkpod</li>
<li>gtranslator</li>
<li>horizon-eda</li>
<li>i3altlayout</li>
<li>inkcut</li>
<li>inkscape-extensions.inkcut</li>
<li>itk</li>
<li>itk4</li>
<li>libgdamm</li>
<li>mirtk</li>
<li>pantheon.elementary-music</li>
<li>python37Packages.cadquery</li>
<li>python37Packages.clifford</li>
<li>python37Packages.dicom2nifti</li>
<li>python37Packages.enamlx</li>
<li>python37Packages.gdcm</li>
<li>python37Packages.mayavi</li>
<li>python37Packages.nixpkgs</li>
<li>python37Packages.pythonocc-core</li>
<li>python37Packages.vtk</li>
<li>python37Packages.vtk_8</li>
<li>python38Packages.clifford</li>
<li>python38Packages.dicom2nifti</li>
<li>python38Packages.enamlx</li>
<li>python38Packages.gdcm</li>
<li>python38Packages.mayavi</li>
<li>python38Packages.nixpkgs</li>
<li>python38Packages.pythonocc-core</li>
<li>python38Packages.vtk</li>
<li>python38Packages.vtk_8</li>
<li>python39Packages.dicom2nifti</li>
<li>python39Packages.enamlx</li>
<li>python39Packages.gdcm</li>
<li>python39Packages.nixpkgs</li>
<li>python39Packages.pythonocc-core</li>
<li>python39Packages.vtk</li>
<li>python39Packages.vtk_8</li>
<li>raiseorlaunch</li>
<li>remarkable-mouse</li>
<li>sequeler</li>
<li>urlwatch</li>
<li>xfce.xfce4-dockbarx-plugin</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108799` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>14 packages failed to build and are new build failures:</summary>
  <ul>
<li>appcsxcad: log was empty</li>
<li>gtkd: log was empty</li>
<li>libsForQt5.qcsxcad: log was empty</li>
<li>(libsForQt515.qcsxcad): log was empty</li>
<li>libsForQt512.qcsxcad: log was empty</li>
<li>libsForQt514.qcsxcad: log was empty</li>
<li>openems: log was empty</li>
<li>python37Packages.python-csxcad: log was empty</li>
<li>python37Packages.python-openems: log was empty</li>
<li>python38Packages.python-csxcad: log was empty</li>
<li>python38Packages.python-openems: log was empty</li>
<li>python39Packages.python-csxcad: log was empty</li>
<li>python39Packages.python-openems: log was empty</li>
<li>tilix: log was empty</li>
  </ul>
</details>
<details>
<summary>
73 packages built:
</summary>
<ul>
<li>ants</li>
<li>c3d</li>
<li>cq-editor</li>
<li>csxcad</li>
<li>datasette (python38Packages.datasette)</li>
<li>dockbarx</li>
<li>elastix</li>
<li>ezminc</li>
<li>gdcm</li>
<li>glom</li>
<li>gnome3.anjuta</li>
<li>libgda (gnome3.libgda)</li>
<li>gtkpod</li>
<li>gtranslator</li>
<li>horizon-eda</li>
<li>i3altlayout</li>
<li>inkcut</li>
<li>inkscape-extensions.inkcut</li>
<li>itk</li>
<li>itk4</li>
<li>libgdamm</li>
<li>mirtk</li>
<li>mitmproxy (python38Packages.mitmproxy)</li>
<li>pantheon.elementary-music</li>
<li>python37Packages.asgiref</li>
<li>python37Packages.cadquery</li>
<li>python37Packages.channels</li>
<li>python37Packages.channels-redis</li>
<li>python37Packages.clifford</li>
<li>python37Packages.daphne</li>
<li>python37Packages.datasette</li>
<li>python37Packages.dicom2nifti</li>
<li>python37Packages.django_3</li>
<li>python37Packages.enamlx</li>
<li>python37Packages.gdcm</li>
<li>python37Packages.mayavi</li>
<li>python37Packages.mitmproxy</li>
<li>python37Packages.nixpkgs</li>
<li>python37Packages.pythonocc-core</li>
<li>python37Packages.vtk (python37Packages.vtk_7)</li>
<li>python37Packages.vtk_8</li>
<li>python38Packages.asgiref</li>
<li>python38Packages.channels</li>
<li>python38Packages.channels-redis</li>
<li>python38Packages.clifford</li>
<li>python38Packages.daphne</li>
<li>python38Packages.dicom2nifti</li>
<li>python38Packages.django_3</li>
<li>python38Packages.enamlx</li>
<li>python38Packages.gdcm</li>
<li>python38Packages.mayavi</li>
<li>python38Packages.nixpkgs</li>
<li>python38Packages.pythonocc-core</li>
<li>python38Packages.vtk (python38Packages.vtk_7)</li>
<li>python38Packages.vtk_8</li>
<li>python39Packages.asgiref</li>
<li>python39Packages.channels</li>
<li>python39Packages.channels-redis</li>
<li>python39Packages.daphne</li>
<li>python39Packages.dicom2nifti</li>
<li>python39Packages.django_3</li>
<li>python39Packages.enamlx</li>
<li>python39Packages.gdcm</li>
<li>python39Packages.mitmproxy</li>
<li>python39Packages.nixpkgs</li>
<li>python39Packages.pythonocc-core</li>
<li>python39Packages.vtk (python39Packages.vtk_7)</li>
<li>python39Packages.vtk_8</li>
<li>raiseorlaunch</li>
<li>remarkable-mouse</li>
<li>sequeler</li>
<li>urlwatch</li>
<li>xfce.xfce4-dockbarx-plugin (xfce.xfce4_dockbarx_plugin)</li>
</ul>
</details>

**Pull request merged**